nature/forms.py

```diff
@@ -12,8 +12,18 @@ class FeatureForm(forms.ModelForm):
     feature_class = forms.ModelChoiceField(
         queryset=FeatureClass.objects.order_by('name'),
         label=FeatureClass._meta.verbose_name.capitalize(),
+        help_text='Mihin kohdeluokkaan kohde kuuluu',
```

Comment:
Maybe also move this to Meta class `help_text`?

Reply:
Does not work, the help text does not get displayed if it is specified in the Meta class. Suggestions welcome.